src/endpoints/getTeamStats.ts

```diff
@@ -85,7 +116,7 @@ export const getTeamStats = (config: HLTVConfig) => async ({
     place: eventEl.find('.statsCenterText').text(),
     event: {
       id: Number(
-        eventEl.find('.image-and-label').first().attr('href')!.split('=')[1]
+        eventEl.find('.image-and-label').first().attr('href')!.split('=')[eventEl.find('.image-and-label').first().attr('href')!.split('=').length - 1]
```

Comment:
What is the reason for this change?

Reply:
For this, when doing the current roster, the `split('=')[1]` is wrong because the url contains 5 = signs because of the `?lineup=` in the url. So, it wasn't finding the event id. This change makes it so it just grabs the id after the last = sign. Does that explanation make sense?